pkg/cvo/status.go

```diff
@@ -443,6 +444,12 @@ func convertErrorToProgressing(history []configv1.UpdateHistory, now time.Time,
 	switch uErr.UpdateEffect {
 	case payload.UpdateEffectNone:
 		return uErr.Reason, fmt.Sprintf("waiting on %s", uErr.Name), true
+	case payload.UpdateEffectReport:
+		if len(uErr.Names) > 0 {
+			return uErr.Reason, fmt.Sprintf("waiting on these operators: %s", strings.Join(uErr.Names, ", ")), false
+		} else {
+			return uErr.Reason, fmt.Sprintf("waiting on operator %s", uErr.Name), false
```

Comment:
we aren't actually waiting in the report case, right?  And if report-level errors are our most serious, we probably completed the update, and want to be `Progressing=False`, with the report-level errors in the `Failing=True` message?

Reply:
Changed them to `degraded...` since for now that's the only time we use that `effect` and I couldn't come up with a good generic message.